ush/find_last_cycle.sh

```diff
@@ -16,7 +16,7 @@
 #
 #   The supported directory structures are:
 #      Operations:  $tankdir/$net/version/$run.$pdy/$hh/atmos/$monitor
-#      Workflow  :  $tankdir/$monitor/stats/$net/$run.$pdy/$hh
+#      Workflow  :  $tankdir/$net/$fun.$pdy/$hh/products/atmos/$monitor
```

Comment:
Should `$fun` be `$run`?

Reply:
Yup that's supposed to be $run.  It's in the intro comments section so it didn't cause a problem in testing.